package.json

```diff
@@ -13,7 +13,8 @@
     "types": "quicktype --src-lang typescript --src ./types/*.ts",
     "prepare:ts": "npm run types -- --lang typescript --out ts/packages/ts/src/types/generated.ts --prefer-unions",
     "prepare:vscode": "npm run types -- --lang schema --out ts/packages/vscode/public/cfgu.schema.json && json -I -f ts/packages/vscode/public/cfgu.schema.json -e 'this.$ref=\"#/definitions/ConfigSchemaContents\"' -e 'this.$schema=undefined'",
-    "prepare:py": "npm run types -- --lang python --python-version 3.7 --nice-property-names --out py/configu/core/generated.py"
+    "prepare:py": "npm run types -- --lang python --python-version 3.7 --nice-property-names --out py/configu/core/generated.py",
+    "prepare:go": "npm run types -- --lang go --package configu --out go/generated.go"
```

Comment:
@jamesL92 Do you believe any specific flags are necessary? you can run `npx quicktype --help` to view the options.